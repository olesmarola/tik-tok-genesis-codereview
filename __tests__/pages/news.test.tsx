import { render } from '@testing-library/react';

import News from '../../pages/news';

describe('News Page', () => {
  it('renders news page unchanged', () => {
    const { container } = render(<News />);

    expect(container).toMatchSnapshot();
  });
});
