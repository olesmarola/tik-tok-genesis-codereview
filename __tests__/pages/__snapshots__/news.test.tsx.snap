// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News Page renders news page unchanged 1`] = `
<div>
  <div
    class="css-1x62r3r-LoaderContainer e1kq1r143"
    data-testid="loader"
  >
    <svg
      class="css-1hvm1xh-LoaderSvg e1kq1r142"
      fill="none"
      height="64"
      viewBox="0 0 24 24"
      width="64"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        class="css-1dkprk2-LoaderCircle e1kq1r141"
        cx="12"
        cy="12"
        r="10"
        stroke="currentColor"
        stroke-width="4"
      />
      <path
        class="css-1rrjpam-LoaderPath e1kq1r140"
        d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
        fill="currentColor"
      />
    </svg>
  </div>
</div>
`;
